conman/routes/tests/test_models.py

```diff
@@ -45,6 +49,16 @@ def test_unique_url(self):
             Route.objects.create(url='/')
 
 
+class RouteCheckTest(TestCase):
+    """Test Route.check()."""
+    def test_defer_to_handler(self):
+        """Route.check returns the result of Route.handler_class().check()."""
+        with mock.patch('conman.routes.models.Route.handler_class') as handler:
+            errors = Route.check()
+
+        self.assertEqual(errors, handler.check(Route))
```

Comment:
Sorry, I edited by original suggestion when I realised `handler` wouldn't be called directly.